datastructs/arrays/SingleDimensionArray.py

```diff
@@ -34,15 +61,27 @@ def __init__(self, elem_type, size, init=[]):
             if len(init) == size:
                 for id, elem in enumerate(init):
                     self._check_value(elem)
-                    self.array[id].data = elem          
+                    self.array[id].data = elem
             else:
                 raise InvalidArrayInitializerSize
-    
+
     def get(self, index):
+        """
+        Gets element at index passed in
+        :param index: index to get element from
+        :return: element at desired index
+        """
         self._check_index(index)
         return self.array[index].data
 
     def update(self, index, value):
+        """
+        Checks index and value passed in for exception errors, then puts value
```

Comment:
How about we rewrite the second part as, "Then updates the array index with the value passed"